### PR TITLE
ci: use tag `latest` for latest releases, `nightly` for nightly versions

### PR DESCRIPTION
**Pull request by ajnavarro**
Added a new tag for them: nightly.

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: .github/workflows/nightlies.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           distribution: goreleaser-pro
           version: latest
```
Comment (thehowl) [inline .github/workflows/nightlies.yml:41]: Should this not be updated?

**Review by thehowl — COMMENTED** [review]
BTW, I think `nightly` is better, `master` if it exists should be the latest commit on master...

File: .github/workflows/nightlies.yml (hunk 1 above)
Comment (ajnavarro) [reply .github/workflows/nightlies.yml]: fixed the version!

**Comment by ajnavarro** [comment]
@thehowl I moved the tag from master to nightly, it makes much more sense.

**Review by thehowl — APPROVED** [review]

**Review by moul — APPROVED** [review]

**Pull request merged**